students/uss_tomasz/lesson_06_dictionaries_tuples_sets_args_kwargs/6_4/guess_the_number.py

```diff
@@ -0,0 +1,20 @@
+def string_of_numbers_to_list_of_int(string_of_numbers):
+    list_of_str = string_of_numbers.split()
+    list_of_int = [int(i) for i in list_of_str]
+    return list_of_int
+
+
+N = 10
+REMAINING_NUMBERS = set(range(1, N + 1))
+AUGUSTUS_SECRET_NUMBER = 6
```

Comment:
consts (`N`, `AUGUSTUS_SECRET_NUMBER`) should rather be at the beginning of the file and should not be modified
variables (`REMAINING_NUMBERS`) should not be capitalized according to pep8